packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/TaskService.java

```diff
@@ -323,6 +323,22 @@ private void onTaskResponseAllInstanceReceive(final Message message) {
 				rLock.unlock();
 			}
 
+			// if the task failed, lets log the stdout / stderr
+			if (resp.getStatus() == TaskStatus.FAILED) {
+				if (resp.getStdout() != null && resp.getStdout().length() > 0) {
+					log.error("Task {} failed, logging stdout", resp.getId());
+					System.out.print(resp.getStdout());
```

Comment:
Is there a reason for having both `log.error` and `System.out`?

Reply:
ohhh

Reply:
The log prefixes it with the time, filename, line etc and then the system out does an unformatted string. Just looks a bit nicer